crypto/crypto.go

```diff
@@ -25,33 +25,29 @@ import (
 	"encoding/pem"
 	"fmt"
 
-	"github.com/onflow/flow-go-sdk/crypto/internal/crypto"
-	"github.com/onflow/flow-go-sdk/crypto/internal/crypto/hash"
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/crypto/hash"
 )
 
 // SignatureAlgorithm is an identifier for a signature algorithm (and parameters if applicable).
-type SignatureAlgorithm int
+type SignatureAlgorithm crypto.SigningAlgorithm
```

Comment:
This kind of duplicated types could be avoided if we don't mind deleting `flow-go-sdk/crypto.SignatureAlgorithm`. That's a breaking change though.

Reply:
I think you can say `type SignatureAlgorithm = crypto.SigningAlgorithm` here

Reply:
It's not possible because of [this function](https://github.com/onflow/flow-go-sdk/blob/tarak/use-flow-go-crypto/crypto/crypto.go#L44) is not allowed to declare function for a receiver from another package. 
The function could be totally deleted if we don't mind removing  `flow-go-sdk/crypto.SignatureAlgorithm`.